internal/browsir.go

```diff
@@ -55,8 +55,21 @@ func (c Command) add(args []string) error {
 	}
 }
 
-func (c Command) rm(args []string) error {
-	return fmt.Errorf("rm not implemented")
+func (c Command) remove(args []string) error {
+	switch args[0] {
+	case "link":
+		fmt.Println("rm link is not yet implemented")
+	case "shortcut":
+		if len(args) < 2 {
+			utils.ExitLog("Please, see --help flag to check usage for add command")
+		}
```

Comment:
Since this is used more than twice we should consider extract into a utility function like:

```
func checkArgs(currentArgs uint8, expectArgs uint8)
```

@404answernotfound This is just a comment and we should consider add this for beginners onboarding. What do you think?

Reply:
I've open a new issue related to this.